tests/image/test_emf.py

```diff
@@ -0,0 +1,40 @@
+# encoding: utf-8
+
+"""Unit test suite for docx.image.emf module"""
+
+from __future__ import absolute_import, print_function
```

Comment:
remove **encoding** and **__future__** lines.
 This project only supports Python3.8+